CollAction/Views/Manage/Index.cshtml

```diff
@@ -10,94 +10,152 @@
+                </div>
+
+            </div>
+        </div>
+
+        <br/><br/>
+
+        <div class="row">
+            <div class="container">
+                <div class="panel panel-default">
+                        <div class="panel-heading"><dt>Newsletter Subscription:</dt></div>
+                        <div class="panel-body">
+                            <div class="row">
+                                <div class="col-md-12">
+                                        <form asp-controller="Manage" asp-action="NewsletterSubscription" method="post" class="form-horizontal">
+                                            <div asp-validation-summary="All" class="text-danger"></div>
+                                            <input type="hidden" asp-for="NewsletterSubscription" value="@(!Model.NewsletterSubscription)" />
+                                            @Html.ValidationMessage("NewsletterSubscription")
+                                            @if (Model.NewsletterSubscription)
+                                            {
+                                                <input type="submit" value="Unsubscribe from newsletter" />
+                                            } else
+                                            {
+                                                <input type="submit" value="Subscribe to newsletter" />
+                                            }
+                                        </form>
+                                    </div>
+                                <div class="col-md-9"></div>
+                            </div>
+                        </div>
+                </div>
+            </div>
+        </div>
+
+        <br/><br/>
+
+        <div class="row">
+            <div class="container">
+                <div class="panel panel-default">
+                        <div class="panel-heading"><dt>Delete Account</dt></div>
+                        <div class="panel-body">
+                            <div class="row">
+                                <div class="alert alert-warning" role="alert"><dt>Warning: This will permanently delete your data!</dt></div>
+                            </div>
+                            <div class="row">
+                                <div class="col-md-3">
+                                        <form asp-controller="Account" asp-action="Delete" method="post">
+                                            <input class="symetric_buttons" type="submit" value="Delete" />
+                                        </form>
+                                    </div>
+                                <div class="col-md-9"></div>
+                            </div>
+                        </div>
+                </div>
+            </div>
+        </div>
+        <br/>
+
+        <div class="row">
+
+            <div class="panel panel-default">
+                    <div class="panel-heading"><dt>Projects You've Subscribed To:</dt></div>
```

Comment:
The indentation seems a bit inconsistent here and there. Maybe run a formatter over this?